src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1798,6 +1798,40 @@
       <summary>Creates an ImplicitStackAllocArrayCreationExpressionSyntax node.</summary>
     </FactoryComment>
   </Node>
+  <Node Name="CollectionCreationExpressionSyntax" Base="ExpressionSyntax">
+    <Kind Name="CollectionCreationExpression"/>
+    <Field Name="OpenBracketToken" Type="SyntaxToken">
+      <Kind Name="OpenBracketToken"/>
+    </Field>
+    <Field Name="Elements" Type="SeparatedSyntaxList&lt;CollectionElementSyntax&gt;" AllowTrailingSeparator="true">
+      <PropertyComment>
+        <summary>SeparatedSyntaxList of CollectionElementSyntax representing the list of elements in the collection expression.</summary>
+      </PropertyComment>
+    </Field>
+    <Field Name="CloseBracketToken" Type="SyntaxToken">
+      <Kind Name="CloseBracketToken"/>
+    </Field>
+  </Node>
+  <AbstractNode Name="CollectionElementSyntax" Base="CSharpSyntaxNode" />
+  <Node Name="ExpressionElementSyntax" Base="CollectionElementSyntax">
+    <Kind Name="ExpressionElement"/>
+    <Field Name="Expression" Type="ExpressionSyntax" />
+  </Node>
+  <Node Name="SpreadElementSyntax" Base="CollectionElementSyntax">
```

Comment:
I think this could be a prefix operator, unless you envision the expression to be optional - the same situation for slice-patterns.

Reply:
Given that it's only for this specific construct, this makes sense to me.  `..e` is not a general construct for other locations.